cmd/kubeadm/app/cmd/config_test.go

```diff
@@ -451,6 +451,32 @@ func TestNewCmdConfigPrintActionDefaults(t *testing.T) {
 			componentConfigs: "KubeProxyConfiguration,KubeletConfiguration",
 			cmdProc:          newCmdConfigPrintJoinDefaults,
 		},
+		{
+			name: "ResetConfiguration: No component configs",
+			expectedKinds: []string{
+				constants.ResetConfigurationKind,
+			},
+			cmdProc: newCmdConfigPrintResetDefaults,
+		},
+		{
+			name: "JoinConfiguration: KubeProxyConfiguration",
+			expectedKinds: []string{
+				"KubeProxyConfiguration",
+				constants.ResetConfigurationKind,
+			},
+			componentConfigs: "KubeProxyConfiguration",
+			cmdProc:          newCmdConfigPrintResetDefaults,
+		},
+		{
+			name: "JoinConfiguration: KubeProxyConfiguration and KubeletConfiguration",
```

Comment:
`ReseteConfiguration: KubeProxyConfiguration and KubeletConfiguration`

Reply:
and thinking this more, is there any case for the resetConfiguration to be configured along with component configs? I don't think so, we should just remove those cases.